doc/rados/operations/monitoring.rst

```diff
@@ -178,37 +178,35 @@ to a healthy state:
     2017-07-25 10:11:13.535493 mon.a mon.0 172.21.9.34:6789/0 110 : cluster [INF] Health check cleared: PG_DEGRADED (was: Degraded data redundancy: 2 pgs unclean, 2 pgs degraded, 2 pgs undersized)
     2017-07-25 10:11:13.535577 mon.a mon.0 172.21.9.34:6789/0 111 : cluster [INF] Cluster is now healthy
 
-
 Network Performance Checks
 --------------------------
 
-Ceph OSDs send heartbeat ping messages amongst themselves to monitor daemon availability.  We
-also use the response times to monitor network performance.
-While it is possible that a busy OSD could delay a ping response, we can assume
-that if a network switch fails multiple delays will be detected between distinct pairs of OSDs.
+Ceph OSDs send heartbeat ping messages to each other in order to  monitor daemon availability 
+and network performance. If a single delayed ping response is detected, this might indicate 
+nothing more than a busy OSD. But if multiple delays between distinct pairs of OSDs are 
```

Comment:
But if delays between multiple pairs of ODSs are reported, there might be a network switch, NIC, or layer 1 failure.